src/app/components/Account/ContractCreatorInfo.tsx

```diff
@@ -48,26 +47,17 @@ export const ContractCreatorInfo: FC<{
   alwaysTrim?: boolean
 }> = ({ scope, isLoading, creationTxHash, alwaysTrim }) => {
   const { t } = useTranslation()
-  const { isMobile } = useScreenSize()
 
   return isLoading ? (
     <Skeleton variant="text" sx={{ width: '50%' }} />
   ) : creationTxHash === undefined ? (
     t('common.missing')
   ) : (
-    <Box
-      sx={{
-        display: 'flex',
-        flexDirection: isMobile ? 'column' : 'row',
-        alignItems: isMobile ? 'flex-start' : 'center',
-        gap: isMobile ? 0 : 2,
-        minWidth: '25%',
-      }}
-    >
+    <>
       <TxSender scope={scope} txHash={creationTxHash} alwaysTrim={alwaysTrim} />
-      <Box>{t('contract.createdAt')}</Box>
+      <Box>&nbsp;{t('contract.createdAt')}&nbsp;</Box>
```

Comment:
I would remove the first &nbsp; as addr has as extra padding anyway (highlighting). on the other hand it doesn't matter as hash is misaligned as well due to MaybeWithTooltip wrapper?

Reply:
I don't think we want highlighting to affect layout with its padding, so I added it in anticipation of fixed highlighting